src/tests/ftest/util/mpio_utils.py

```diff
@@ -103,10 +103,10 @@ def run_romio(self, basepath, hostlist, romio_test_repo):
             raise MpioFailed("<ROMIO Test FAILED> \nException occurred: {}"
                              .format(str(excep)))
 
-    def run_llnl_mpi4py(self, basepath, hostfile, pool_uuid, test_repo,
-                        test_name, client_processes):
+    def run_llnl_mpi4py_hdf5(self, basepath, hostfile, pool_uuid, test_repo,
```

Comment:
(pylint-too-many-arguments) Too many arguments (7/5)

Reply:
either add a pylint comment to disable or fix